lib/Dialect/Torch/IR/TorchOps.cpp

```diff
@@ -1529,6 +1529,40 @@ OpFoldResult Aten__Contains__StrOp::fold(ArrayRef<Attribute> operands) {
   return nullptr;
 }
 
+//===----------------------------------------------------------------------===//
+// Aten__Contains__IntListOp
+//===----------------------------------------------------------------------===//
+
+static bool isListConstructNotModified(Value torchList) {
+  if (!llvm::all_of(torchList.getUsers(), [](Operation *op) {
+        return isa<Aten__Contains__IntListOp>(op);
+      }))
+    return false;
+
+  return torchList.getDefiningOp<Torch::PrimListConstructOp>();
```

Comment:
This should not compile. `getDefiningOp` returns an `Operation *`, not a `bool`. Please make sure things compile.